src/webgl/p5.Shader.js

```diff
@@ -1012,6 +1012,10 @@ p5.Shader = class Shader {
     return this.uniforms.uStrokeWeight !== undefined;
   }
 
+  isImageShader() {
```

Comment:
I think we don't need this method because we don't want our `imageShader(yourShader)` method to work depending on some uniforms (`uImageSampler` for your case). We could probably take this one out.